frontend/src/components/UserUpdateRoles.vue

```diff
@@ -221,9 +227,12 @@ export default {
 </script>
 
 <style>
+.right-gutters .col {
+  padding: 10px 12px 10px 0px;
```

Comment:
Very minor but "padding: 10px 12px 10px 0px;" could be "padding: 10px 12px 10px 0;" (like the other 0's are handled below).